tools/CustomTasks/NuSpecs/PublishNupkg.cmd

```diff
@@ -2,10 +2,10 @@
 SETLOCAL
 set ERRORLEVEL=
 
-for %%A IN (%~dp0MUXCustomBuildTasks.??.??.??.nupkg) do (
+for %%A IN (%~dp0MUXCustomBuildTasks.??.??.*.nupkg) do (
     echo Candidate nuget package: %%A
-    echo %~dp0..\..\NugetWrapper.cmd push %%A -Source https://pkgs.dev.azure.com/WinUI/_packaging/MUXControls.Private/nuget/v3/index.json -apikey VSTS
-    %~dp0..\..\NugetWrapper.cmd push %%A -Source https://pkgs.dev.azure.com/WinUI/_packaging/MUXControls.Private/nuget/v3/index.json -apikey VSTS
+    echo %~dp0..\..\NugetWrapper.cmd push %%A -Source https://microsoft.pkgs.visualstudio.com/DefaultCollection/_packaging/DEPControls/nuget/v3/index.json -apikey VSTS
```

Comment:
>DEPControls [](start = 123, length = 11)

Any reason that it's not MUXControls? Maybe we need create a dummy MUXControls under it

Reply:
The internal nuget feed is named depcontrols. It is internal only, so there is no real need to rename it to MUXControls.

We don't consume any packages directly from this feed (since it is not accessible publicly). The depcontrols feed is just there for staging purposes. We ultimately publish the packages to dotnetfeed which is available publicly.